_config.yml

```
@@ -17,7 +17,15 @@ languages:
     name: Afrikaans
     title: Woordelys
     blurb: >
-      In translation.
+      `glosario` is 'n oop bron woordelys van terminologie wat in datawetenskap 
+      gebruik word en beskikbaar gestel word aanlyn sowel as 'n programmateek in 
+      [R](https://github.com/carpentries/glosario-r/) en [Python]
+      (https://github.com/carpentries/glosario-py/). Deur woordelyssleutels by 'n
```

Markdown will not interpret the Python link as a link unless the square brackets and round brackets are on the same line. 

```suggestion
      [R](https://github.com/carpentries/glosario-r/) en [Python](https://github.com/carpentries/glosario-py/). 
      Deur woordelyssleutels by 'n
```

---

There should be a button above that says "commit suggestion" you can use that to accept the suggestion without changing it on your account if that makes things easier :)